### PR TITLE
chore: added injury effects table

### LINKED ISSUE #183: Two Missing tables from the beta rule book.
**Is your feature request related to a problem? Please describe.**  
We have an injury duration table but not their effects table or the shardblade injury table

image
image

**Describe the solution you'd like**  
add those tables

**Version**  
0.2.1

**Checklist:**
- [ ] I have searched the existing issues to ensure this feature has not already been requested.

### PR DESCRIPTION
**Pull request by Doc-Sun**
**Type**  
What type of pull request is this? (e.g., Bug fix, Feature, Refactor, etc.)

- [ ] Bug fix
- [x] Feature
- [ ] Refactor
- [ ] Other (please describe):

**Description**  
Added Missing tables from the beta rules that were missing that could be used for rolling or choosing.

**Related Issue**  
 Half closes Closes #183

**How Has This Been Tested?**  
Tested and built in foundry, exported .json and added to repo

**Checklist:**  
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes do not introduce any new warnings or errors.
- [x] My PR does not contain any copyrighted works that I do not have permission to use.
- [x] I have tested my changes on Foundry VTT version: [12.331].

**Additional context**  
_Add any other context or information here that would be useful for reviewers._

### CODE REVIEW

**Review by MangoFVTT — CHANGES_REQUESTED** [review]
Shardblade injuries are a Stormlight exclusive and should definitely be moved to a module (probably to the playtest materials for now). The other one can stay 👍 We'll need to think of styling for these as well I suspect, in order to be able to apply the effect directly from the chat cards

**Comment by Doc-Sun** [comment]
> Shardblade injuries are a Stormlight exclusive and should definitely be moved to a module (probably to the playtest materials for now). The other one can stay 👍 We'll need to think of styling for these as well I suspect, in order to be able to apply the effect directly from the chat cards

Thanks. Removed the shardblades chart from this PR and renamed the PR

File: src/packs/tables/injury-effects.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    }
+  },
+  "_stats": {
+    "coreVersion": "12.331",
+    "systemId": "cosmere-rpg",
+    "systemVersion": "0.2.1",
+    "createdTime": 1732504183989,
+    "modifiedTime": 1732507522837,
+    "lastModifiedBy": "mlhqK8ft6P9hbaYE"
+  },
```
Comment (MangoFVTT) [inline src/packs/tables/injury-effects.json:109]: These stats should also be removed

**Review by MangoFVTT — CHANGES_REQUESTED** [review]

File: src/packs/tables/injury-effects.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    {
+      "type": "text",
+      "weight": 1,
+      "range": [
+        1,
+        2
+      ],
+      "drawn": false,
+      "_id": "gpJNDSRsoaWUM0T0",
+      "text": "<strong>Exausted(-1)</strong> Any Injury that lowers your overall stamina.",
```
[the remark this answers is not in the document]
Comment (Doc-Sun) [reply src/packs/tables/injury-effects.json]: fixed in latest push

File: src/packs/tables/injury-effects.json [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    {
+      "type": "text",
+      "weight": 1,
+      "range": [
+        3,
+        3
+      ],
+      "drawn": false,
+      "_id": "hp5hWMYB10llrOtH",
+      "text": "<strong>Exausted(-2)</strong> Any Injury that lowers your overall stamina.",
```
[the remark this answers is not in the document]
Comment (Doc-Sun) [reply src/packs/tables/injury-effects.json]: fixed in latest push

File: src/packs/tables/injury-effects.json [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  "displayRoll": true,
+  "folder": null,
+  "flags": {
+    "exportSource": {
+      "world": "b9test",
+      "system": "cosmere-rpg",
+      "coreVersion": "12.331",
+      "systemVersion": "0.2.1"
+    }
+  },
```
[the remark this answers is not in the document]
Comment (Doc-Sun) [reply src/packs/tables/injury-effects.json]: fixed in latest push

File: src/packs/tables/injury-effects.json (hunk 1 above)
Comment (Doc-Sun) [reply src/packs/tables/injury-effects.json]: fixed in latest push

File: src/packs/tables/injury-effects.json [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+      "img": "icons/svg/cancel.svg",
+      "documentId": null,
+      "flags": {}
+    }
+  ],
+  "replacement": true,
+  "displayRoll": true,
+  "folder": null,
+  "flags": {},
+  "_stats": {},
```
Comment (MangoFVTT) [inline src/packs/tables/injury-effects.json:95]: I believe _stats has to go completely, sorry should've clarified

Comment (Doc-Sun) [reply src/packs/tables/injury-effects.json]: fixed and pushed

File: src/packs/tables/injury-effects.json [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    {
+      "type": "text",
+      "weight": 1,
+      "range": [
+        7,
+        7
+      ],
+      "drawn": false,
+      "_id": "Xsus0oW97bVGoZEW",
+      "text": "<strong>Supprised</strong> Overwhelmed due to the shock of an injury.",
```
Comment (zithith) [inline src/packs/tables/injury-effects.json:71]: Typo in Surprised

**Comment by zithith** [comment]
Would be kinda good to have the conditions somewhere as Items in the system so that these could have documentIds, but that's a thought for another day

**Comment by Doc-Sun** [comment]
> Would be kinda good to have the conditions somewhere as Items in the system so that these could have documentIds, but that's a thought for another day

That would be nice (id really like them to be like the pf2e conditions that even show up in the top right of the screen with hover over text when placed on a actor token and that token is selected) but thats a little beyond me for today. This I can do

File: src/packs/tables/injury-effects.json (hunk 6 above)
Comment (Doc-Sun) [reply src/packs/tables/injury-effects.json]: fixed in push

**Review by MangoFVTT — DISMISSED** [review]

**Review by MangoFVTT — APPROVED** [review]

**Review by zithith — APPROVED** [review]

**Pull request merged**